src/components/Logo/Logo.story.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import Logo from './Logo';
+
+const stories = storiesOf('Branding', module);
+
+stories.add('Logo', () => (
+  <Logo
+    kind={select('Kind', ['logo mark', 'full'], 'full')}
+    size={select('Size', ['small', 'medium', 'large'], 'medium')}
```

Comment:
need to add dimensions to these since they document size, `small (3rem|3px)` etc